test/e2e/run_test.sh

```diff
@@ -226,6 +241,21 @@ while (( "$#" )); do
       SKIP_BUILD=true
       shift
     ;;
+    --use-cluster)
+      if [ -n "$2" ] && [[ "$2" != --* ]]; then
+        NAME="$2"
+        USE_EXISTING_CLUSTER=true
```

Comment:
No `USE_EXISTING_CLUSTER` is needed. If the user wants to  use an exisiting cluster, just specify `--skip-setup`